resources/tenants/controller/tenants.go

```diff
@@ -15,25 +15,37 @@
 package controller
 
 import (
-	"net/http"
-
 	"github.com/ant0ine/go-json-rest/rest"
+	deploymentsController "github.com/mendersoftware/deployments/resources/deployments/controller"
+	"github.com/mendersoftware/deployments/resources/tenants/model"
+	"github.com/mendersoftware/go-lib-micro/identity"
 	"github.com/mendersoftware/go-lib-micro/log"
 	"github.com/mendersoftware/go-lib-micro/rest_utils"
-
-	"github.com/mendersoftware/deployments/resources/tenants/model"
+	"net/http"
 )
 
 type Controller struct {
 	model model.Model
 }
 
+type DeploymentsControllerWithTenant struct {
+	model                 model.Model
+	deploymentsController deploymentsController.DeploymentsController
```

Comment:
tenant controller with deployments controller?
is this the only way?
it looks really weird to me

Reply:
I can give this a shot:

https://docs.mongodb.com/manual/reference/method/db.collection.count/

Reply:
shouldn't that be model ?